test/index.spec.js

```diff
@@ -460,6 +460,16 @@ describe('helpers', () => {
         port: '1234'
       })
     })
+
+    it('returns a node friendly address with dns', () => {
+      expect(
+        multiaddr('/dns4/wss0.bootstrap.libp2p.io/tcp/443').nodeAddress()
+      ).to.be.eql({
+        address: 'wss0.bootstrap.libp2p.io',
+        family: 'IPv4',
+        port: '443'
+      })
+    })
   })
 
```

Comment:
Please also add a test case for checking if the multiaddr is missing the prefix with `/`